Handling CephClusterReadOnlyMode (#23)
red-hat-storage/ocs-sop: sop/modules/OSD/partials/troubleshoot_ceph_err.adoc

The first two status commands provide the overall cluster health. The normal state for cluster operations is HEALTH_OK, but will still function when the state is in a HEALTH_WARN state. If you are in a WARN state, then the cluster is in a condition that it may enter the HEALTH_ERROR state at which point all disk I/O operations are halted. If a HEALTH_WARN state is observed, then one should take action to prevent the cluster from halting when it enters the HEALTH_ERROR state.

Problem 1::

> Alert Manager shows that the OSD is full

Reviewer: ```suggestion
Ceph status shows that the OSD is full
```

Author: Would ceph status show that osd is full? last we checked it did not right?

Reviewer: Yeah, it will show OSDs full when it crosses 85%

> 1) Check the alert manager for readonly alert

Reviewer: ```suggestion
1) Check the alert manager for CephClusterReadOnly alert
```

> ----

Reviewer: add [source,role="execute"] before this line

curl -k -H "Authorization: Bearer $(oc -n openshift-monitoring sa get-token prometheus-k8s)"  https://${MYALERTMANAGER}/api/v1/alerts | jq '.data[] | select( .labels.alertname) | { ALERT: .labels.alertname, STATE: .status.state}' 
----

> 2) if the above curl command shows that the ceph cluster is read only mode, then see :

Reviewer: ```suggestion
2) If CephClusterReadOnly alert is listed from the above curl command, then see :
```

> ----

Reviewer: add `[source,role="execute"]` before this line
